libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -131,6 +131,15 @@ const AP_Param::GroupInfo AP_AHRS::var_info[] = {
     AP_GROUPEND
 };
 
+// return a smoothed and corrected gyro vector using the latest ins data (which may not have been consumed by the EKF yet)
+Vector3f AP_AHRS::get_gyro_latest(void) const
+{
+    // check new method of getting gyro values
+    uint8_t primary_gyro = get_ins().get_primary_gyro();
```

Comment:
I think this should use `get_primary_IMU_index`, otherwise the bias it is adding in the next line might be for the wrong gyro.
That method though returns the primary accel when DCM is being used - which doesn't happen for Copter, but it is something to be aware of since this method can be used by any vehicle. I'm not sure if in practice it ever happens that primary gyro and accel are different, but it is theoretically possible.